apps/api/src/persistence/migrations/01/remove-base-digital-asset-url.migration.spec.ts

```diff
@@ -0,0 +1,222 @@
+import createTestModule from '../../../app/controllers/__tests__/createTestModule';
```

Comment:
We need to opt out of running `.migration.spec` tests as part of quality-checks. The migration test should be run manually only while working on the  migration. It's likely that future model updates will break an old migration test. We **do not** want to maintain these tests once the migration has been applied to the production database(s)!